README.md

```diff
@@ -27,8 +27,9 @@ Spawns multiple synchronized SSH sessions inside a tmux session.
 
 Options:
   -h                  Show help
-  -n <name>           Name of the tmux session (default: cssh)
+  -n <name>           Name of the tmux session or window (default: cssh)
```

Comment:
also, is it possible to occupy the current window instead of creating a new one?

Reply:
just a thought, this isn't critical of course

Reply:
imitate the same behavior of creating of new session i create a new window, but imho you are right is much more intuitive if use the current window. When i have free time i'll check how good is it.